src/components/library/SeriesTrackerModal.tsx

```diff
@@ -177,7 +181,11 @@ const SeriesTrackerModal: React.FC<Props> = (props: Props) => {
     );
   };
 
-  const renderTrackEntry = (trackerId: string) => {
+  const renderTrackEntry = (
+    trackerId: string,
+    trackerUrl: string,
+    trackerName: string
+  ) => {
```

Comment:
nitpick: could probably just use `trackerMetadata: TrackerMetadata` at this point instead

Reply:
Forgot to mention type `TrackerMetadata` & hence it wasn't detecting it correctly. I thought something was wrong 😂